plutus-use-cases/test/Spec/Auction.hs

```diff
@@ -364,7 +364,7 @@ tests =
             .&&. walletFundsChange w3 mempty)
             auctionTrace2
         , testProperty "QuickCheck property" $
-            withMaxSuccess 10 prop_FinishAuction
+            withMaxSuccess 100 prop_FinishAuction
         , testProperty "NLFP fails" $
```

Comment:
Might be worth while adding `prop_Auction` here for good measure.

Reply:
(it currently fails)

Reply:
Also, 100 tests are not generally enough to get non-flaky tests.

Reply:
@ak3n Have you tried this out?

Reply:
Oh right, that was the bug filed here https://github.com/input-output-hk/plutus-apps/issues/735

Reply:
@ak3n I'm starting to suspect the failure was not due to `calculateDelta` but to the tx size restrictions. It passes on this branch for me too.